src/actions/VaccineActions.js

```diff
@@ -169,33 +193,43 @@ const startDownloadAllLogs = macAddress => async (dispatch, getState) => {
   return null;
 };
 
-const setLogInterval = (macAddress, interval) => async dispatch => {
-  let ok = false;
-  let error = `Sensor response was not equal to 'Interval: ${interval}s'`;
-
+const setLogInterval = (macAddress, interval) => dispatch => {
   dispatch(setLogIntervalStart(macAddress));
+  const regex = new RegExp(`Interval: ${interval}s`); // TODO: update with sensor specific response as needed
+  const error = `Sensor response was not equal to 'Interval: ${interval}s'`;
+
+  return BleService()
+    .updateLogInterval(macAddress, interval)
```

Comment:
Changing to await not needed, but I think it would be good to use `logIntervalWithRetries` - it really helps when constantly getting disconnected with the poorer sensors.

Reply:
await is needed as I'm checking the response, which itself isn't super necessary, but I think that its nice to know